base_jsonify/models/models.py

```diff
@@ -71,6 +71,10 @@ def jsonify(self, parser):
                     value = rec[field_name]
                     if value is False and field_type != 'boolean':
                         value = None
+                    elif field_type == "date":
+                        value = fields.Date.to_string(value)
+                    elif field_type == "datetime":
+                        value = fields.Datetime.to_string(value)
```

Comment:
I think it's better to be compliant with ISO 8601 here.
Date is ok,  but date_time defined here https://github.com/odoo/odoo/blob/12.0/odoo/tools/misc.py#L568
is with a space as separator between dates and time. Plus it's lacking timezone.

https://www.w3.org/TR/NOTE-datetime

Reply:
@lmignon have you seen this comment? Sounds important to me too.